src/pages/Dashboard/Modals/BorrowRepay/Repay/styles.ts

```diff
@@ -0,0 +1,26 @@
+import { css } from '@emotion/react';
+import { useTheme } from '@mui/material';
+
+export const useStyles = () => {
+  const theme = useTheme();
+
+  return {
+    walletBalance: css`
+      margin-bottom: ${theme.spacing(3)};
+    `,
+    selectButtonsContainer: css`
+      display: flex;
+      flex-direction: row;
+    `,
+    selectButton: css`
+      flex: 1;
+      border-radius: ${theme.spacing(13)};
```

Comment:
shouldn't we have shape for this case?

Reply:
These buttons are the only ones I can see in the app using this border radius, so I figured perhaps it would make more sense to use `theme.spacing` directly. Do you think we should add this to the shape?